plugins/module_utils/job.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) IBM Corporation 2019 - 2023
+# Copyright (c) IBM Corporation 2019 - 2024
```

Comment:
seems like we've been preferring to use commas for copyright: `2019, 2024`

Reply:
Done